components/ClubPage/SocialLink.js

```diff
@@ -0,0 +1,11 @@
+export default props => {
+  let club = props.club;
```

Comment:
I think const is better than let here. Also, you can do `const { club, item, type: social } = props` (or `social: type`, I don't remember exactly lol

Also, the if-else can be simplified to
```
if (social == "email") {
    return club[item.name];
  } 
  return <a href={club[item.name]}>{club.name}</a>;
```

Reply:
Or even better, you can do 
```
export default ({ club, item, social: type }) => {
if (social == "email") {
    return club[item.name];
  } 
 return <a href={club[item.name]}>{club.name}</a>;
}
```